<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogPcParms</class>
 <widget class="QDialog" name="dialogPcParms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialogPcParms</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Color balance options  (changes slow for large datasets)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="balanceOffButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No color balance</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="balanceOnButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram equalization independently on each channel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>False color options</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>May be slow for large datasets</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="falseNaturalButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Natural (green-white)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="falseBlueButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue-to-red</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="elevAbsButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Absolute elevation</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="elevRelButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevation rel to smoothed DEM</string>
   </property>
  </widget>
  <widget class="QSlider" name="minHueSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="maxHueSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="minHueEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxHueEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elev of min hue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elev o max hue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>250</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply new color scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>balanceOffButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doBalanceOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balanceOnButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doBalanceOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>falseNaturalButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doNatural()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>falseBlueButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doBlue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minHueSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doMinHue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxHueSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doMaxHue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>elevRelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doElevRel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>elevAbsButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doElevAbs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>doNewFalse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogPcParms</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doBalanceOff()</slot>
  <slot>doBalanceOn()</slot>
  <slot>doNatural()</slot>
  <slot>doBlue()</slot>
  <slot>doMinHue()</slot>
  <slot>doMaxHue()</slot>
  <slot>doElevRel()</slot>
  <slot>doElevAbs()</slot>
  <slot>doNewFalse()</slot>
 </slots>
</ui>
